From 45da720c44c7a7758cc811bea609116088168f25 Mon Sep 17 00:00:00 2001
From: Huisong Li <lihuisong@huawei.com>
Date: Wed, 31 Mar 2021 18:01:35 +0800
Subject: [PATCH 077/189] net/hns3: fix some function names for copper media
 type

PHY is a common concept for the copper and optical media type interface.
There are some inappropriate function names for copper ports, which
needs to be adjusted.

Fixes: 2e4859f3b362 ("net/hns3: support PF device with copper PHYs")

Signed-off-by: Huisong Li <lihuisong@huawei.com>
Signed-off-by: Min Hu (Connor) <humin29@huawei.com>
---
 drivers/net/hns3/hns3_ethdev.c | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/drivers/net/hns3/hns3_ethdev.c b/drivers/net/hns3/hns3_ethdev.c
index 05f199e..4dff016 100644
--- a/drivers/net/hns3/hns3_ethdev.c
+++ b/drivers/net/hns3/hns3_ethdev.c
@@ -4597,7 +4597,7 @@ hns3_update_fiber_link_info(struct hns3_hw *hw)
 }
 
 static void
-hns3_parse_phy_params(struct hns3_cmd_desc *desc, struct hns3_mac *mac)
+hns3_parse_copper_phy_params(struct hns3_cmd_desc *desc, struct hns3_mac *mac)
 {
 	struct hns3_phy_params_bd0_cmd *req;
 
@@ -4615,7 +4615,7 @@ hns3_parse_phy_params(struct hns3_cmd_desc *desc, struct hns3_mac *mac)
 }
 
 static int
-hns3_get_phy_params(struct hns3_hw *hw, struct hns3_mac *mac)
+hns3_get_copper_phy_params(struct hns3_hw *hw, struct hns3_mac *mac)
 {
 	struct hns3_cmd_desc desc[HNS3_PHY_PARAM_CFG_BD_NUM];
 	uint16_t i;
@@ -4634,20 +4634,20 @@ hns3_get_phy_params(struct hns3_hw *hw, struct hns3_mac *mac)
 		return ret;
 	}
 
-	hns3_parse_phy_params(desc, mac);
+	hns3_parse_copper_phy_params(desc, mac);
 
 	return 0;
 }
 
 static int
-hns3_update_phy_link_info(struct hns3_hw *hw)
+hns3_update_copper_link_info(struct hns3_hw *hw)
 {
 	struct hns3_mac *mac = &hw->mac;
 	struct hns3_mac mac_info;
 	int ret;
 
 	memset(&mac_info, 0, sizeof(struct hns3_mac));
-	ret = hns3_get_phy_params(hw, &mac_info);
+	ret = hns3_get_copper_phy_params(hw, &mac_info);
 	if (ret)
 		return ret;
 
@@ -4676,7 +4676,7 @@ hns3_update_link_info(struct rte_eth_dev *eth_dev)
 	int ret = 0;
 
 	if (hw->mac.media_type == HNS3_MEDIA_TYPE_COPPER)
-		ret = hns3_update_phy_link_info(hw);
+		ret = hns3_update_copper_link_info(hw);
 	else if (hw->mac.media_type == HNS3_MEDIA_TYPE_FIBER)
 		ret = hns3_update_fiber_link_info(hw);
 
-- 
2.7.4

